NativeApp/src/components/Login/Form/index.js

```diff
@@ -38,45 +38,68 @@ class LoginForm extends Component {
     const { handleLogin, hasError, loading } = this.props;
     const { changeColor } = this;
     return (
-      <View>
-        <TextInput
-          id="input-1"
-          onFocus={() => changeColor('input-1', '#00DCFA')}
-          onBlur={() => changeColor('input-1', 'gray')}
-          placeholder="Email"
-          onChangeText={text => this.setState({ email: text })}
-          value={email}
-          autoCapitalize="none"
-          underlineColorAndroid="transparent"
-          selectionColor="#00DCFA"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor1 }}
-        />
-        <TextInput
-          id="input-2"
-          onFocus={() => changeColor('input-2', '#00DCFA')}
-          onBlur={() => changeColor('input-2', 'gray')}
-          placeholder="Password"
-          onChangeText={text => this.setState({ password: text })}
-          value={password}
-          secureTextEntry
-          underlineColorAndroid="transparent"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor2 }}
-          selectionColor="#00DCFA"
-        />
-        {
-          hasError && (
-            <Caption type="base" style={styles.red}>
-              Incorrect email or password
-            </Caption>
-          )
-        }
+      <View style={styles.view}>
+        <View>
+          <FormLabel>Email</FormLabel>
+          <FormInput
+            id="input-1"
+            onChangeText={text => this.setState({ email: text })}
+            inputStyle={{
```

Comment:
I would move the inputStyles into StyleSheets below then import using `styles.inputStyles`